Category-Colors/category-colors
# Feature/slider number input

# Description
Issue # (add the issue number here, if applicable)



Describe what you're adding

- Added slider and input number state
- Added event handler for the slider
- Added increment/decrement functions to increase/decrease the number value state
- We can now update the number state by using the slider
- Both the slider and number input are set up so that they both update the shared 'number' value state



# QA
List out how to test your changes and add a screenshot of what it should look like, when applicable.
Please see the screenshot below: 

![sliderinput](https://github.com/Category-Colors/category-colors/assets/57161327/1d3e6b11-d737-4f88-bd1e-1b0f6d1d86cd)
